test/e2e/e2e_test.go

```diff
@@ -28,8 +29,114 @@ import (
 
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	ecv1alpha1 "go.etcd.io/etcd-operator/api/v1alpha1"
 )
 
+var etcdVersion = os.Getenv("ETCD_VERSION")
+
+// TestZeroMemberCluster tests if zero member Etcd Cluster does not create its StatefulSet
```

Comment:
Could you add a comment something like: we will update this test once https://github.com/etcd-io/etcd-operator/issues/125 is resolved.